corehq/apps/cleanup/management/commands/reprocess_incomplete_submissions.py

```diff
@@ -89,7 +89,11 @@ def handle(self, domain, **options):
                 page = paginator.page(page_number)
                 for stub in page.object_list:
                     result = reprocess_unfinished_stub(stub, save=not dryrun)
-                    if result:
+                    if not result:
+                        logger.info("Processing skipped")
+                    elif result.error:
+                        logger.info(f"Form re-processed failed: {stub.xform_id}: {result.error}")
+                    else:
                         cases = ', '.join([c.case_id for c in result.cases])
                         ledgers = ', '.join([l.ledger_reference for l in result.ledgers])
                         logger.info("Form re-processed successfully: {}:{}".format(
```

Comment:
I know this isn't part of your PR, but is this calling `format` on a string with too many arguments? It looks like we may need to add more `{}` to the string template.

Reply:
or just remove the extra arguments